firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/TaskUtils.java

```diff
@@ -35,4 +36,29 @@
     }
     return task;
   }
+
```

Comment:
This is nice! we can use this to cleanup other checks like this in a bunch of places as we encounter them in future CLs)